lib/state.rb

```diff
@@ -51,6 +51,12 @@ def deployers_for_today
     Deployers.for_today
   end
 
+  def hotfix_in_progress?
+    !GitHub.client.branch(GitHub::GITHUB_REPO, GitHub::HOTFIX_BRANCH).nil?
```

Comment:
At this point I think I'd include `ActiveSupport` and call `.present?`?

Reply:
yes! that's wrote I wrote originally before remembering its not core ruby :)